Sources/App/Core/Query+Support/JoinedQueryBuilder.swift

```diff
@@ -24,6 +24,7 @@ struct JoinedQueryBuilder<J: ModelInitializable> {
         return self
     }
 
+    // periphery:ignore
```

Comment:
Leaving these unused ones in here, because they're tricky to write and handy to have around when we write new queries. It's something you don't want to have to do when you're working on a query.